[cairomm] static build error

### Operating system

x64-windows

### Compiler

MSVC 19.35.32217.1

### Steps to reproduce the behavior

```Shell
`vcpkg --triplet x64-windows-static install gtkmm`
```


### Failure logs

```
-- Downloading https://www.cairographics.org/releases/cairomm-1.16.2.tar.xz -> cairomm-1.16.2.tar.xz...
-- Extracting source D:/dev/vcpkg/downloads/cairomm-1.16.2.tar.xz
-- Applying patch fix_include_path.patch
-- Using source at D:/dev/vcpkg/buildtrees/cairomm/src/cairomm-1-486fec65f2.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for x64-windows-static
-- Downloading https://www.python.org/ftp/python/3.10.7/python-3.10.7-embed-amd64.zip -> python-3.10.7-embed-amd64.zip...
-- Configuring x64-windows-static-dbg
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst -> msys-mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst...
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst -> msys-mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst...
-- Using msys root at D:/dev/vcpkg/downloads/tools/msys2/9a1ec3f33446b195
-- Configuring x64-windows-static-dbg done
-- Configuring x64-windows-static-rel
-- Configuring x64-windows-static-rel done
-- Package x64-windows-static-dbg
-- Package x64-windows-static-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" install -v
    Working Directory: D:/dev/vcpkg/buildtrees/cairomm/x64-windows-static-rel
    Error code: 1
    See logs for more information:
      D:\dev\vcpkg\buildtrees\cairomm\package-x64-windows-static-rel-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_meson.cmake:33 (vcpkg_execute_required_process)
  ports/cairomm/portfile.cmake:22 (vcpkg_install_meson)
  scripts/ports.cmake:147 (include)



```
<details><summary>D:\dev\vcpkg\buildtrees\cairomm\package-x64-windows-static-rel-out.log</summary>

```
[1/39] "C:/Program Files (x86)/Windows Kits/10/bin/10.0.22000.0/x64/rc.exe" "/nologo" "/focairomm/MSVC_NMake_cairomm_cairomm.rc_cairomm.res" "MSVC_NMake/cairomm/cairomm.rc"
[2/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/cl.exe" "-Icairomm\libcairomm-1.16.a.p" "-I." "-I..\src\cairomm-1-486fec65f2.clean" "-ID:/dev/vcpkg/installed/x64-windows-static/include" "-ID:/dev/vcpkg/installed/x64-windows-static/include/cairo" "-ID:/dev/vcpkg/installed/x64-windows-static/include/libpng16" "-ID:/dev/vcpkg/installed/x64-windows-static/include/pixman-1" "/MT" "/nologo" "/showIncludes" "/utf-8" "/W2" "/EHsc" "/std:c++17" "/permissive-" "/Od" "-D_USE_MATH_DEFINES" "/wd4267" "/wd4800" "/utf-8" "-nologo" "-DWIN32" "-D_WINDOWS" "-W3" "-utf-8" "-GR" "-EHsc" "-MP" "-MT" "-O2" "-Oi" "-Gy" "-DNDEBUG" "-Z7" "-DCAIROMM_BUILD=1" "/Fdcairomm\libcairomm-1.16.a.p\quartz_font.cc.pdb" /Focairomm/libcairomm-1.16.a.p/quartz_font.cc.obj "/c" ../src/cairomm-1-486fec65f2.clean/cairomm/quartz_font.cc
cl : Ligne de commande warning D9025 : substitution de '/W2' par '/W3'
cl : Ligne de commande warning D9025 : substitution de '/Od' par '/O2'
[3/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/cl.exe" "-Icairomm\libcairomm-1.16.a.p" "-I." "-I..\src\cairomm-1-486fec65f2.clean" "-ID:/dev/vcpkg/installed/x64-windows-static/include" "-ID:/dev/vcpkg/installed/x64-windows-static/include/cairo" "-ID:/dev/vcpkg/installed/x64-windows-static/include/libpng16" "-ID:/dev/vcpkg/installed/x64-windows-static/include/pixman-1" "/MT" "/nologo" "/showIncludes" "/utf-8" "/W2" "/EHsc" "/std:c++17" "/permissive-" "/Od" "-D_USE_MATH_DEFINES" "/wd4267" "/wd4800" "/utf-8" "-nologo" "-DWIN32" "-D_WINDOWS" "-W3" "-utf-8" "-GR" "-EHsc" "-MP" "-MT" "-O2" "-Oi" "-Gy" "-DNDEBUG" "-Z7" "-DCAIROMM_BUILD=1" "/Fdcairomm\libcairomm-1.16.a.p\fontoptions.cc.pdb" /Focairomm/libcairomm-1.16.a.p/fontoptions.cc.obj "/c" ../src/cairomm-1-486fec65f2.clean/cairomm/fontoptions.cc
cl : Ligne de commande warning D9025 : substitution de '/W2' par '/W3'
cl : Ligne de commande warning D9025 : substitution de '/Od' par '/O2'
[4/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/cl.exe" "-Icairomm\libcairomm-1.16.a.p" "-I." "-I..\src\cairomm-1-486fec65f2.clean" "-ID:/dev/vcpkg/installed/x64-windows-static/include" "-ID:/dev/vcpkg/installed/x64-windows-static/include/cairo" "-ID:/dev/vcpkg/installed/x64-windows-static/include/libpng16" "-ID:/dev/vcpkg/installed/x64-windows-static/include/pixman-1" "/MT" "/nologo" "/showIncludes" "/utf-8" "/W2" "/EHsc" "/std:c++17" "/permissive-" "/Od" "-D_USE_MATH_DEFINES" "/wd4267" "/wd4800" "/utf-8" "-nologo" "-DWIN32" "-D_WINDOWS" "-W3" "-utf-8" "-GR" "-EHsc" "-MP" "-MT" "-O2" "-Oi" "-Gy" "-DNDEBUG" "-Z7" "-DCAIROMM_BUILD=1" "/Fdcairomm\libcairomm-1.16.a.p\matrix.cc.pdb" /Focairomm/libcairomm-1.16.a.p/matrix.cc.obj "/c" ../src/cairomm-1-486fec65f2.clean/cairomm/matrix.cc
...
Skipped 165 lines
...

test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_fill_rule(enum Cairo::Context::FillRule)" (__imp_?set_fill_rule@Context@Cairo@@QEAAXW4FillRule@12@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_fill_rule::test_method(void)" (?test_method@test_fill_rule@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_line_width(double)" (__imp_?set_line_width@Context@Cairo@@QEAAXN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_width::test_method(void)" (?test_method@test_line_width@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_line_cap(enum Cairo::Context::LineCap)" (__imp_?set_line_cap@Context@Cairo@@QEAAXW4LineCap@12@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_cap::test_method(void)" (?test_method@test_line_cap@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_line_join(enum Cairo::Context::LineJoin)" (__imp_?set_line_join@Context@Cairo@@QEAAXW4LineJoin@12@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_join::test_method(void)" (?test_method@test_line_join@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_dash(class std::valarray<double> const &,double)" (__imp_?set_dash@Context@Cairo@@QEAAXAEBV?$valarray@N@std@@N@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_dashes::test_method(void)" (?test_method@test_dashes@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_dash(class std::vector<double,class std::allocator<double> > const &,double)" (__imp_?set_dash@Context@Cairo@@QEAAXAEBV?$vector@NV?$allocator@N@std@@@std@@N@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_dashes::test_method(void)" (?test_method@test_dashes@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::unset_dash(void)" (__imp_?unset_dash@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_dashes::test_method(void)" (?test_method@test_dashes@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_miter_limit(double)" (__imp_?set_miter_limit@Context@Cairo@@QEAAXN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_miter_limit::test_method(void)" (?test_method@test_miter_limit@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::translate(double,double)" (__imp_?translate@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_user_device::test_method(void)" (?test_method@test_user_device@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::scale(double,double)" (__imp_?scale@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_user_device::test_method(void)" (?test_method@test_user_device@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::transform(class Cairo::Matrix const &)" (__imp_?transform@Context@Cairo@@QEAAXAEBVMatrix@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_matrix::test_method(void)" (?test_method@test_matrix@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_matrix(class Cairo::Matrix const &)" (__imp_?set_matrix@Context@Cairo@@QEAAXAEBVMatrix@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_matrix::test_method(void)" (?test_method@test_matrix@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_identity_matrix(void)" (__imp_?set_identity_matrix@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_matrix::test_method(void)" (?test_method@test_matrix@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::user_to_device(double &,double &)const " (__imp_?user_to_device@Context@Cairo@@QEBAXAEAN0@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_user_device::test_method(void)" (?test_method@test_user_device@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::device_to_user(double &,double &)const " (__imp_?device_to_user@Context@Cairo@@QEBAXAEAN0@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_user_device::test_method(void)" (?test_method@test_user_device@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::begin_new_path(void)" (__imp_?begin_new_path@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::move_to(double,double)" (__imp_?move_to@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_current_point::test_method(void)" (?test_method@test_current_point@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::line_to(double,double)" (__imp_?line_to@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::curve_to(double,double,double,double,double,double)" (__imp_?curve_to@Context@Cairo@@QEAAXNNNNNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::arc(double,double,double,double,double)" (__imp_?arc@Context@Cairo@@QEAAXNNNNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::arc_negative(double,double,double,double,double)" (__imp_?arc_negative@Context@Cairo@@QEAAXNNNNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::rel_move_to(double,double)" (__imp_?rel_move_to@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::rel_line_to(double,double)" (__imp_?rel_line_to@Context@Cairo@@QEAAXNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::rel_curve_to(double,double,double,double,double,double)" (__imp_?rel_curve_to@Context@Cairo@@QEAAXNNNNNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::rectangle(double,double,double,double)" (__imp_?rectangle@Context@Cairo@@QEAAXNNNN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_clip::test_method(void)" (?test_method@test_clip@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::close_path(void)" (__imp_?close_path@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::paint(void)" (__imp_?paint@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::stroke(void)" (__imp_?stroke@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_draw::test_method(void)" (?test_method@test_draw@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::clip(void)" (__imp_?clip@Context@Cairo@@QEAAXXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_clip::test_method(void)" (?test_method@test_clip@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::get_clip_extents(double &,double &,double &,double &)const " (__imp_?get_clip_extents@Context@Cairo@@QEBAXAEAN000@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_clip::test_method(void)" (?test_method@test_clip@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_font_options(class Cairo::FontOptions const &)" (__imp_?set_font_options@Context@Cairo@@QEAAXAEBVFontOptions@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_font_options::test_method(void)" (?test_method@test_font_options@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::get_font_options(class Cairo::FontOptions &)const " (__imp_?get_font_options@Context@Cairo@@QEBAXAEAVFontOptions@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_font_options::test_method(void)" (?test_method@test_font_options@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::set_scaled_font(class std::shared_ptr<class Cairo::ScaledFont const > const &)" (__imp_?set_scaled_font@Context@Cairo@@QEAAXAEBV?$shared_ptr@$$CBVScaledFont@Cairo@@@std@@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_scaled_font::test_method(void)" (?test_method@test_scaled_font@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::ScaledFont> __cdecl Cairo::Context::get_scaled_font(void)" (__imp_?get_scaled_font@Context@Cairo@@QEAA?AV?$shared_ptr@VScaledFont@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_scaled_font::test_method(void)" (?test_method@test_scaled_font@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: enum Cairo::Context::Operator __cdecl Cairo::Context::get_operator(void)const " (__imp_?get_operator@Context@Cairo@@QEBA?AW4Operator@12@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_operator::test_method(void)" (?test_method@test_operator@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::Pattern> __cdecl Cairo::Context::get_source(void)" (__imp_?get_source@Context@Cairo@@QEAA?AV?$shared_ptr@VPattern@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_source::test_method(void)" (?test_method@test_source@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::Pattern const > __cdecl Cairo::Context::get_source(void)const " (__imp_?get_source@Context@Cairo@@QEBA?AV?$shared_ptr@$$CBVPattern@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_source::test_method(void)" (?test_method@test_source@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::SurfacePattern> __cdecl Cairo::Context::get_source_for_surface(void)" (__imp_?get_source_for_surface@Context@Cairo@@QEAA?AV?$shared_ptr@VSurfacePattern@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_source::test_method(void)" (?test_method@test_source@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::SurfacePattern const > __cdecl Cairo::Context::get_source_for_surface(void)const " (__imp_?get_source_for_surface@Context@Cairo@@QEBA?AV?$shared_ptr@$$CBVSurfacePattern@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_source::test_method(void)" (?test_method@test_source@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: double __cdecl Cairo::Context::get_tolerance(void)const " (__imp_?get_tolerance@Context@Cairo@@QEBANXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_tolerance::test_method(void)" (?test_method@test_tolerance@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: enum Cairo::Antialias __cdecl Cairo::Context::get_antialias(void)const " (__imp_?get_antialias@Context@Cairo@@QEBA?AW4Antialias@2@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_antialias::test_method(void)" (?test_method@test_antialias@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::get_current_point(double &,double &)const " (__imp_?get_current_point@Context@Cairo@@QEBAXAEAN0@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_current_point::test_method(void)" (?test_method@test_current_point@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: enum Cairo::Context::FillRule __cdecl Cairo::Context::get_fill_rule(void)const " (__imp_?get_fill_rule@Context@Cairo@@QEBA?AW4FillRule@12@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_fill_rule::test_method(void)" (?test_method@test_fill_rule@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: double __cdecl Cairo::Context::get_line_width(void)const " (__imp_?get_line_width@Context@Cairo@@QEBANXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_width::test_method(void)" (?test_method@test_line_width@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: enum Cairo::Context::LineCap __cdecl Cairo::Context::get_line_cap(void)const " (__imp_?get_line_cap@Context@Cairo@@QEBA?AW4LineCap@12@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_cap::test_method(void)" (?test_method@test_line_cap@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: enum Cairo::Context::LineJoin __cdecl Cairo::Context::get_line_join(void)const " (__imp_?get_line_join@Context@Cairo@@QEBA?AW4LineJoin@12@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_line_join::test_method(void)" (?test_method@test_line_join@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: double __cdecl Cairo::Context::get_miter_limit(void)const " (__imp_?get_miter_limit@Context@Cairo@@QEBANXZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_miter_limit::test_method(void)" (?test_method@test_miter_limit@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::get_dash(class std::vector<double,class std::allocator<double> > &,double &)const " (__imp_?get_dash@Context@Cairo@@QEBAXAEAV?$vector@NV?$allocator@N@std@@@std@@AEAN@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_dashes::test_method(void)" (?test_method@test_dashes@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::Context::get_matrix(class Cairo::Matrix &)" (__imp_?get_matrix@Context@Cairo@@QEAAXAEAVMatrix@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_Context::test_matrix::test_method(void)" (?test_method@test_matrix@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class Cairo::Matrix __cdecl Cairo::Context::get_matrix(void)const " (__imp_?get_matrix@Context@Cairo@@QEBA?AVMatrix@2@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_matrix::test_method(void)" (?test_method@test_matrix@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::Surface> __cdecl Cairo::Context::get_target(void)" (__imp_?get_target@Context@Cairo@@QEAA?AV?$shared_ptr@VSurface@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_target::test_method(void)" (?test_method@test_target@Cairo_Context@@QEAAXXZ)
test-context.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::Surface const > __cdecl Cairo::Context::get_target(void)const " (__imp_?get_target@Context@Cairo@@QEBA?AV?$shared_ptr@$$CBVSurface@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_Context::test_target::test_method(void)" (?test_method@test_target@Cairo_Context@@QEAAXXZ)

tests\test_context.exe : fatal error LNK1120: 72 externes non résolus

[36/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/link.exe"  /MACHINE:x64 /OUT:tests/test_scaled_font.exe tests/test_scaled_font.exe.p/test-scaled-font.cc.obj "-INCREMENTAL:NO" "/nologo" "/release" "/nologo" "-machine:x64" "-nologo" "-DEBUG" "-OPT:REF" "-OPT:ICF" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "cairomm\libcairomm-1.16.a" "D:/dev/vcpkg/installed/x64-windows-static/lib/sigc-3.0.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/cairo.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/fontconfig.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libexpatMT.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/freetype.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/bz2.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libpng16.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/zlib.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlidec-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlicommon-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pixman-1.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pthreadVC3.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "C:/local/boost_1_67_0/lib64-msvc-14.1/libboost_unit_test_framework-vc141-mt-s-x64-1_67.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "/SUBSYSTEM:CONSOLE" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
FAILED: tests/test_scaled_font.exe 
"C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/link.exe"  /MACHINE:x64 /OUT:tests/test_scaled_font.exe tests/test_scaled_font.exe.p/test-scaled-font.cc.obj "-INCREMENTAL:NO" "/nologo" "/release" "/nologo" "-machine:x64" "-nologo" "-DEBUG" "-OPT:REF" "-OPT:ICF" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "cairomm\libcairomm-1.16.a" "D:/dev/vcpkg/installed/x64-windows-static/lib/sigc-3.0.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/cairo.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/fontconfig.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libexpatMT.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/freetype.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/bz2.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libpng16.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/zlib.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlidec-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlicommon-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pixman-1.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pthreadVC3.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "C:/local/boost_1_67_0/lib64-msvc-14.1/libboost_unit_test_framework-vc141-mt-s-x64-1_67.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "/SUBSYSTEM:CONSOLE" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
   Création de la bibliothèque tests\test_scaled_font.lib et de l'objet tests\test_scaled_font.exp
LINK : warning LNK4217: le symbole '??0FontOptions@Cairo@@QEAA@XZ (public: __cdecl Cairo::FontOptions::FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-scaled-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_ScaledFont::test_construction::test_method(void)" (?test_method@test_construction@Cairo_ScaledFont@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '??1FontOptions@Cairo@@UEAA@XZ (public: virtual __cdecl Cairo::FontOptions::~FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-scaled-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_ScaledFont::test_construction::test_method(void)" (?test_method@test_construction@Cairo_ScaledFont@@QEAAXXZ)'
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: static class std::shared_ptr<class Cairo::ToyFontFace> __cdecl Cairo::ToyFontFace::create(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,enum Cairo::ToyFontFace::Slant,enum Cairo::ToyFontFace::Weight)" (__imp_?create@ToyFontFace@Cairo@@SA?AV?$shared_ptr@VToyFontFace@Cairo@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@4@W4Slant@12@W4Weight@12@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_construction::test_method(void)" (?test_method@test_construction@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: static class std::shared_ptr<class Cairo::FtFontFace> __cdecl Cairo::FtFontFace::create(struct _FcPattern *)" (__imp_?create@FtFontFace@Cairo@@SA?AV?$shared_ptr@VFtFontFace@Cairo@@@std@@PEAU_FcPattern@@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_ft_scaled_font::test_method(void)" (?test_method@test_ft_scaled_font@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: __cdecl Cairo::Matrix::Matrix(void)" (__imp_??0Matrix@Cairo@@QEAA@XZ) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_construction::test_method(void)" (?test_method@test_construction@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) class Cairo::Matrix __cdecl Cairo::identity_matrix(void)" (__imp_?identity_matrix@Cairo@@YA?AVMatrix@1@XZ) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_ft_scaled_font::test_method(void)" (?test_method@test_ft_scaled_font@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: static class std::shared_ptr<class Cairo::ScaledFont> __cdecl Cairo::ScaledFont::create(class std::shared_ptr<class Cairo::FontFace> const &,class Cairo::Matrix const &,class Cairo::Matrix const &,class Cairo::FontOptions const &)" (__imp_?create@ScaledFont@Cairo@@SA?AV?$shared_ptr@VScaledFont@Cairo@@@std@@AEBV?$shared_ptr@VFontFace@Cairo@@@4@AEBVMatrix@2@1AEBVFontOptions@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_construction::test_method(void)" (?test_method@test_construction@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: class std::shared_ptr<class Cairo::FontFace> __cdecl Cairo::ScaledFont::get_font_face(void)const " (__imp_?get_font_face@ScaledFont@Cairo@@QEBA?AV?$shared_ptr@VFontFace@Cairo@@@std@@XZ) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_get_font_face::test_method(void)" (?test_method@test_get_font_face@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::ScaledFont::text_to_glyphs(double,double,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::vector<struct cairo_glyph_t,class std::allocator<struct cairo_glyph_t> > &,class std::vector<struct cairo_text_cluster_t,class std::allocator<struct cairo_text_cluster_t> > &,enum Cairo::TextClusterFlags &)" (__imp_?text_to_glyphs@ScaledFont@Cairo@@QEAAXNNAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEAV?$vector@Ucairo_glyph_t@@V?$allocator@Ucairo_glyph_t@@@std@@@4@AEAV?$vector@Ucairo_text_cluster_t@@V?$allocator@Ucairo_text_cluster_t@@@std@@@4@AEAW4TextClusterFlags@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_text_to_glyphs::test_method(void)" (?test_method@test_text_to_glyphs@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: void __cdecl Cairo::ScaledFont::get_scale_matrix(class Cairo::Matrix &)const " (__imp_?get_scale_matrix@ScaledFont@Cairo@@QEBAXAEAVMatrix@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_scale_matrix::test_method(void)" (?test_method@test_scale_matrix@Cairo_ScaledFont@@QEAAXXZ)
test-scaled-font.cc.obj : error LNK2019: symbole externe non résolu "__declspec(dllimport) public: static class std::shared_ptr<class Cairo::FtScaledFont> __cdecl Cairo::FtScaledFont::create(class std::shared_ptr<class Cairo::FtFontFace> const &,class Cairo::Matrix const &,class Cairo::Matrix const &,class Cairo::FontOptions const &)" (__imp_?create@FtScaledFont@Cairo@@SA?AV?$shared_ptr@VFtScaledFont@Cairo@@@std@@AEBV?$shared_ptr@VFtFontFace@Cairo@@@4@AEBVMatrix@2@1AEBVFontOptions@2@@Z) référencé dans la fonction "public: void __cdecl Cairo_ScaledFont::test_ft_scaled_font::test_method(void)" (?test_method@test_ft_scaled_font@Cairo_ScaledFont@@QEAAXXZ)

tests\test_scaled_font.exe : fatal error LNK1120: 9 externes non résolus

[37/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/link.exe"  /MACHINE:x64 /OUT:tests/test_font_options.exe tests/test_font_options.exe.p/test-font-options.cc.obj "-INCREMENTAL:NO" "/nologo" "/release" "/nologo" "-machine:x64" "-nologo" "-DEBUG" "-OPT:REF" "-OPT:ICF" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "cairomm\libcairomm-1.16.a" "D:/dev/vcpkg/installed/x64-windows-static/lib/sigc-3.0.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/cairo.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/fontconfig.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libexpatMT.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/freetype.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/bz2.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libpng16.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/zlib.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlidec-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlicommon-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pixman-1.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pthreadVC3.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "C:/local/boost_1_67_0/lib64-msvc-14.1/libboost_unit_test_framework-vc141-mt-s-x64-1_67.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "/SUBSYSTEM:CONSOLE" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
   Création de la bibliothèque tests\test_font_options.lib et de l'objet tests\test_font_options.exp
LINK : warning LNK4217: le symbole '??0FontOptions@Cairo@@QEAA@XZ (public: __cdecl Cairo::FontOptions::FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '??1FontOptions@Cairo@@UEAA@XZ (public: virtual __cdecl Cairo::FontOptions::~FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?merge@FontOptions@Cairo@@QEAAXAEBV12@@Z (public: void __cdecl Cairo::FontOptions::merge(class Cairo::FontOptions const &))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?hash@FontOptions@Cairo@@QEBAKXZ (public: unsigned long __cdecl Cairo::FontOptions::hash(void)const )' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?set_antialias@FontOptions@Cairo@@QEAAXW4Antialias@2@@Z (public: void __cdecl Cairo::FontOptions::set_antialias(enum Cairo::Antialias))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?get_antialias@FontOptions@Cairo@@QEBA?AW4Antialias@2@XZ (public: enum Cairo::Antialias __cdecl Cairo::FontOptions::get_antialias(void)const )' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?set_subpixel_order@FontOptions@Cairo@@QEAAXW4SubpixelOrder@2@@Z (public: void __cdecl Cairo::FontOptions::set_subpixel_order(enum Cairo::SubpixelOrder))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?get_subpixel_order@FontOptions@Cairo@@QEBA?AW4SubpixelOrder@2@XZ (public: enum Cairo::SubpixelOrder __cdecl Cairo::FontOptions::get_subpixel_order(void)const )' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?set_hint_style@FontOptions@Cairo@@QEAAXW4HintStyle@12@@Z (public: void __cdecl Cairo::FontOptions::set_hint_style(enum Cairo::FontOptions::HintStyle))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?get_hint_style@FontOptions@Cairo@@QEBA?AW4HintStyle@12@XZ (public: enum Cairo::FontOptions::HintStyle __cdecl Cairo::FontOptions::get_hint_style(void)const )' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?set_hint_metrics@FontOptions@Cairo@@QEAAXW4HintMetrics@12@@Z (public: void __cdecl Cairo::FontOptions::set_hint_metrics(enum Cairo::FontOptions::HintMetrics))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'

LINK : warning LNK4217: le symbole '?get_hint_metrics@FontOptions@Cairo@@QEBA?AW4HintMetrics@12@XZ (public: enum Cairo::FontOptions::HintMetrics __cdecl Cairo::FontOptions::get_hint_metrics(void)const )' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-font-options.cc.obj' dans la fonction '"public: void __cdecl Cairo_FontOptions::test_excercise::test_method(void)" (?test_method@test_excercise@Cairo_FontOptions@@QEAAXXZ)'

[38/39] "C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/link.exe"  /MACHINE:x64 /OUT:tests/test_user_font.exe tests/test_user_font.exe.p/test-user-font.cc.obj "-INCREMENTAL:NO" "/nologo" "/release" "/nologo" "-machine:x64" "-nologo" "-DEBUG" "-OPT:REF" "-OPT:ICF" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "cairomm\libcairomm-1.16.a" "D:/dev/vcpkg/installed/x64-windows-static/lib/sigc-3.0.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/cairo.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/fontconfig.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libexpatMT.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/freetype.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/bz2.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/libpng16.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/zlib.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlidec-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/brotlicommon-static.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pixman-1.lib" "D:/dev/vcpkg/installed/x64-windows-static/lib/pthreadVC3.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "C:/local/boost_1_67_0/lib64-msvc-14.1/libboost_unit_test_framework-vc141-mt-s-x64-1_67.lib" "/LIBPATH:D:/dev/vcpkg/installed/x64-windows-static/lib" "gdi32.lib" "msimg32.lib" "dwrite.lib" "d2d1.lib" "windowscodecs.lib" "/SUBSYSTEM:CONSOLE" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
   Création de la bibliothèque tests\test_user_font.lib et de l'objet tests\test_user_font.exp
LINK : warning LNK4217: le symbole '?get_status@FontFace@Cairo@@QEBA?AW4_cairo_status@@XZ (public: enum _cairo_status __cdecl Cairo::FontFace::get_status(void)const )' défini dans 'libcairomm-1.16.a(fontface.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '??1UserFontFace@Cairo@@UEAA@XZ (public: virtual __cdecl Cairo::UserFontFace::~UserFontFace(void))' défini dans 'libcairomm-1.16.a(fontface.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: virtual __cdecl Cairo_UserFontFace::ExceptionUserFont::~ExceptionUserFont(void)" (??1ExceptionUserFont@Cairo_UserFontFace@@UEAA@XZ)'
LINK : warning LNK4217: le symbole '??0UserFontFace@Cairo@@IEAA@XZ (protected: __cdecl Cairo::UserFontFace::UserFontFace(void))' défini dans 'libcairomm-1.16.a(fontface.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"protected: __cdecl Cairo_UserFontFace::ExceptionUserFont::ExceptionUserFont(int)" (??0ExceptionUserFont@Cairo_UserFontFace@@IEAA@H@Z)'
LINK : warning LNK4217: le symbole '??0FontOptions@Cairo@@QEAA@XZ (public: __cdecl Cairo::FontOptions::FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '??1FontOptions@Cairo@@UEAA@XZ (public: virtual __cdecl Cairo::FontOptions::~FontOptions(void))' défini dans 'libcairomm-1.16.a(fontoptions.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?identity_matrix@Cairo@@YA?AVMatrix@1@XZ (class Cairo::Matrix __cdecl Cairo::identity_matrix(void))' défini dans 'libcairomm-1.16.a(matrix.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?scaling_matrix@Cairo@@YA?AVMatrix@1@NN@Z (class Cairo::Matrix __cdecl Cairo::scaling_matrix(double,double))' défini dans 'libcairomm-1.16.a(matrix.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?create@ScaledFont@Cairo@@SA?AV?$shared_ptr@VScaledFont@Cairo@@@std@@AEBV?$shared_ptr@VFontFace@Cairo@@@4@AEBVMatrix@2@1AEBVFontOptions@2@@Z (public: static class std::shared_ptr<class Cairo::ScaledFont> __cdecl Cairo::ScaledFont::create(class std::shared_ptr<class Cairo::FontFace> const &,class Cairo::Matrix const &,class Cairo::Matrix const &,class Cairo::FontOptions const &))' défini dans 'libcairomm-1.16.a(scaledfont.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_user_font_exception::test_method(void)" (?test_method@test_user_font_exception@Cairo_UserFontFace@@QEAAXXZ)'
LINK : warning LNK4217: le symbole '?create@ImageSurface@Cairo@@SA?AV?$shared_ptr@VImageSurface@Cairo@@@std@@W4Format@Surface@2@HH@Z (public: static class std::shared_ptr<class Cairo::ImageSurface> __cdecl Cairo::ImageSurface::create(enum Cairo::Surface::Format,int,int))' défini dans 'libcairomm-1.16.a(surface.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: __cdecl TestSetup::TestSetup(void)" (??0TestSetup@@QEAA@XZ)'
LINK : warning LNK4217: le symbole '?create@Context@Cairo@@SA?AV?$shared_ptr@VContext@Cairo@@@std@@AEBV?$shared_ptr@VSurface@Cairo@@@4@@Z (public: static class std::shared_ptr<class Cairo::Context> __cdecl Cairo::Context::create(class std::shared_ptr<class Cairo::Surface> const &))' défini dans 'libcairomm-1.16.a(context.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: __cdecl TestSetup::TestSetup(void)" (??0TestSetup@@QEAA@XZ)'
LINK : warning LNK4217: le symbole '?show_text@Context@Cairo@@QEAAXAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z (public: void __cdecl Cairo::Context::show_text(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &))' défini dans 'libcairomm-1.16.a(context.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_implement_both::test_method(void)" (?test_method@test_implement_both@Cairo_UserFontFace@@QEAAXXZ)'

LINK : warning LNK4217: le symbole '?set_font_face@Context@Cairo@@QEAAXAEBV?$shared_ptr@$$CBVFontFace@Cairo@@@std@@@Z (public: void __cdecl Cairo::Context::set_font_face(class std::shared_ptr<class Cairo::FontFace const > const &))' défini dans 'libcairomm-1.16.a(context.cc.obj)' est importé par 'test-user-font.cc.obj' dans la fonction '"public: void __cdecl Cairo_UserFontFace::test_implement_both::test_method(void)" (?test_method@test_implement_both@Cairo_UserFontFace@@QEAAXXZ)'

ninja: build stopped: subcommand failed.
```
</details>

### Additional context

It would seem that some cairomm test are trying to link cairo dynamically while the triplet is set to static.

--- a/ports/cairomm/portfile.cmake
+++ b/ports/cairomm/portfile.cmake
@@ -17,6 +17,7 @@ vcpkg_configure_meson(
     OPTIONS
         -Dbuild-examples=false
         -Dmsvc14x-parallel-installable=false    # Use separate DLL and LIB filenames for Visual Studio 2017 and 2019
+        -Dbuild-tests=false
 )
 
 vcpkg_install_meson()

--- a/ports/cairomm/vcpkg.json
+++ b/ports/cairomm/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cairomm",
   "version": "1.16.2",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A C++ wrapper for the cairo graphics library",
   "homepage": "https://www.cairographics.org",
   "license": "LGPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1354,7 +1354,7 @@
     },
     "cairomm": {
       "baseline": "1.16.2",
-      "port-version": 2
+      "port-version": 3
     },
     "calceph": {
       "baseline": "3.5.2",

--- a/versions/c-/cairomm.json
+++ b/versions/c-/cairomm.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "7a7e6f9ecf73d0a2ec27e9a70a099746e643ec84",
+      "version": "1.16.2",
+      "port-version": 3
+    },
+    {
       "git-tree": "9e3f77f8b4d02d2a5bd898b0ca96031da57aae12",
       "version": "1.16.2",
       "port-version": 2

